vdr package uses some macros like HOST_NAME_MAX, NAME_MAX, which are defined
in limits.h and needs to be explicitly included on ppc64le.
----
  
--- vdr-2.3.8/config.h
+++ vdr-2.3.8/config.h
@@ -19,6 +19,9 @@
 #include "i18n.h"
 #include "font.h"
 #include "tools.h"
+#ifdef __powerpc64__
+#include <limits.h> 
+#endif
 
 // VDR's own version number:
 
--- vdr-2.3.8/tools.c
+++ vdr-2.3.8/tools.c
@@ -27,6 +27,9 @@
 #include <utime.h>
 #include "i18n.h"
 #include "thread.h"
+#ifdef __powerpc64__
+#include <limits.h>
+#endif
 
 int SysLogLevel = 3;
 
